config/webpack.prod.config.js

```diff
@@ -51,6 +51,11 @@ module.exports = Merge.smart(commonConfig, {
   },
   plugins: [
     new ExtractTextPlugin('studio-frontend.min.css'),
+    new webpack.DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify('production'),
```

Comment:
If `NODE_ENV=production` is set in the production build task (looks like it is), can you pull in `process.env.NODE_ENV` here instead of hardcoding the value?